server/plugin/api.go

```diff
@@ -719,8 +719,8 @@ func (p *Plugin) createIncident(w http.ResponseWriter, r *http.Request) {
 	}
 
 	userID := r.Header.Get(constants.HeaderMattermostUserID)
-	permissionStatusCode, channelType, permissionErr := p.HasChannelPermissions(userID, incident.ChannelID, false)
-	if permissionErr != nil {
+	permissionStatusCode, permissionErr := p.HasChannelPermissions(userID, incident.ChannelID)
+	if permissionErr != nil && permissionErr.Error() != constants.ErrorInvalidChannelType {
```

Comment:
How are we handling the case when this permissionErr is equal to `constants.ErrorInvalidChannelType`?